sormas-app/app/src/androidTest/java/de/symeda/sormas/app/CaseBackendTest.java

```diff
@@ -420,19 +427,23 @@ public void testEditCasePermissionWhenRegionMatch() throws DaoException, SQLExce
 	}
 
 	@Test
-	public void testEditCasePermissionWhenRegionDoesNotMatchAndCaseNotCreatedByUser() throws DaoException, SQLException {
+	public void testEditCasePermissionWhenDistrictDoesNotMatchAndCaseNotCreatedByUser() throws DaoException, SQLException {
 		Case caze = TestEntityCreator.createCase();
-		caze.setRegion(null);
-		caze.setDistrict(null);
+		caze.setRegion(DatabaseHelper.getRegionDao().queryUuid(TestHelper.REGION_UUID));
+		District secondDistrict = DatabaseHelper.getDistrictDao().queryUuid(TestHelper.SECOND_DISTRICT_UUID);
+		caze.setDistrict(secondDistrict);
+		caze.setResponsibleDistrict(secondDistrict);
 		caze.setHealthFacility(null);
 
-		UserRole userRole = UserRole.HOSPITAL_INFORMANT;
```

Comment:
Not sure why it's necessary to switch this - should also work with the HOSPITAL_INFORMANT